plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java

```diff
@@ -87,6 +98,33 @@ public final void testKeywordsDefined() {
   }
   
   @Test
+  public final void testBuildProperties() throws IOException {
+    String[] binIncludes = buildProperties.get("bin.includes").split(",\\s*");
+    Set<String> includes = Sets.newHashSet(binIncludes);
+    
+    Assert.assertTrue(includes.contains("plugin.xml"));
+    Assert.assertTrue(includes.contains("plugin.properties"));
+    Assert.assertTrue(includes.contains("."));
+    Assert.assertTrue(includes.contains("META-INF/"));
+    
+    testIncludedIfPresent(includes, "helpContexts.xml");
+    testIncludedIfPresent(includes, "icons/");
+    testIncludedIfPresent(includes, "lib/");
+    testIncludedIfPresent(includes, "README.md/");
+    testIncludedIfPresent(includes, "epl-v10.html/");
```

Comment:
`epl-v10.html` is a file, not a directory.

Checking for these files is expanding the scope of what I'd expect from`BasePluginXmlTests`, but I really like the idea — it's like a PDE-lint.  We could do some more sophisticated checking.  For example, if `epl-v10.html` is found, ensure it has a [`Bundle-License` header](https://www.osgi.org/bundle-headers-reference/) pointing to `https://www.eclipse.org/legal/epl-v10.html`.  If it has a `Bundle-License`, ensure it points to a valid library.